index.html

```diff
@@ -394,49 +395,49 @@ <h4>LET'S CONNECT</h4>
       <div class="overlay">
         <!-- <img src="images/contactformshapes.svg" alt="footershapes" /> -->
 
-        <div class="content"> 
-        <article class="footer-text">
-          <h2>Contact me</h2>
-          <p>
-            If you have an application you are interested in developing, a
-            feature that you need built or a project that needs coding. I’d love
-            to help with it
-          </p>
-        </article>
+        <div class="content">
+          <article class="footer-text">
+            <h2>Contact me</h2>
+            <p>
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it
+            </p>
+          </article>
+
```

Comment:
- [ ] Please remove the empty gap between the about and contact section. The gap is too much some users can think that the page ends after about section and they don't reach to contact section.

### See the image for reference 

![image](https://user-images.githubusercontent.com/83803414/166641291-eca2d3e5-3878-4a12-b6f2-0483ae9bf4c3.png)